hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/CryptoCreateHandler.java

```diff
@@ -128,7 +127,7 @@ public void pureChecks(@NonNull final TransactionBody txn) throws PreCheckExcept
             validateTruePreCheck(op.realmIDOrThrow().realmNum() == 0, INVALID_ACCOUNT_ID);
         }
         // HIP 904 now allows for unlimited auto-associations
-        validateTruePreCheck(op.maxAutomaticTokenAssociations() >= -1, INVALID_TRANSACTION_BODY);
+        validateTruePreCheck(op.maxAutomaticTokenAssociations() >= -1, INVALID_MAX_AUTO_ASSOCIATIONS);
```

Comment:
We can probably use `UNLIMITED_AUTOMATIC_ASSOCIATIONS` everywhere we use -1

Reply:
Done.